lib/routes/society/namespace.ts

```diff
@@ -0,0 +1,11 @@
+import type { Namespace } from '@/types';
```

Comment:
Please [reuse current namespace instead of creating variations for the same namespace](https://docs.rsshub.app/joinus/new-rss/start-code).
https://github.com/DIYgod/RSSHub/blob/fcd4b8f8f6162ecebc424047f75d28274a17b25e/lib/routes/shu/namespace.ts#L1-L8

Reply:
Thanks a lot. I did not notice it belongs to SHU before.